xc/locals.tf

```diff
@@ -3,7 +3,7 @@ locals {
   build_suffix = data.tfe_outputs.infra.values.build_suffix
   origin_bigip = try(data.tfe_outputs.bigip[0].values.bigip_public_vip, "")
   origin_nginx = try(data.tfe_outputs.nap[0].values.external_name, data.tfe_outputs.nic[0].values.external_name, "")
-  origin_server = "${coalesce(local.origin_bigip, local.origin_nginx, var.serviceName)}"
+  origin_server = "${coalesce(local.origin_nginx, local.origin_bigip, var.serviceName)}"
```

Comment:
since this use case has both bigip and nginx, if you use bigip first, CICD is failing so changed the code little bit @knowbase